pkg/build/build.go

```diff
@@ -122,10 +132,13 @@ func (b *Builder) copyCombustionFile(scriptSubDir string, scriptName string) err
 		return fmt.Errorf("writing file: %w", err)
 	}
 
-	// Keep a running list of all added combustion files. When we add the combustion
+	return nil
+}
+
+func (b *Builder) registerCombustionScript(scriptName string) {
+	// Keep a running list of all added combustion scripts. When we add the combustion
 	// "script" file (the one Combustion itself looks at), we'll concatenate calls to
 	// each of these to that script.
-	b.combustionScripts = append(b.combustionScripts, scriptName)
```

Comment:
Removing this from the default behavior was intentional. We're not going to necessarily want every file in the combustion directory added and attempted to be executed by script, so I took this opportunity to pull it out into its own call.